lux-lib/src/build/mod.rs

```diff
@@ -55,6 +55,8 @@ pub struct Build<'a, R: Rockspec + HasIntegrity> {
     #[builder(start_fn)]
     tree: &'a Tree,
     #[builder(start_fn)]
+    is_entrypoint: bool,
+    #[builder(start_fn)]
```

Comment:
suggestion(non-blocking): we probably want to make this an enum

Reply:
Yeah, makes sense.
I've added

```rust
struct EntryType {
  Entrypoint,
  DependencyOnly,
}
```

to the `tree` module.